src/test/java/com/integrivideo/Data.java

```diff
@@ -20,6 +20,8 @@
     public final static String IMAGE_URL_4K = "http://4k.com/wp-content/uploads/2014/06/4k-image-tiger-jumping.jpg";
     public final static String INSTALLED_CHAT_WITH_DOMAIN = "http://dmitryrak11.000webhostapp.com/COMPONENT_CHAT_WITH_DOMAIN/";
     public final static String LOGIN_PAGE_URL = BASE_URL + "login";
+    public final static String MSG_WITH_URL = "Hi, check this site: http://bbc.com";
```

Comment:
You can use variable URL_BBC_SITE  here or event delete this variable too and user just URL_BBC_SITE  in the test